redbot/core/dev_commands.py

```diff
@@ -135,14 +137,18 @@ async def debug(self, ctx, *, code):
         lines or asynchronous code, see [p]repl or [p]eval.
 
         Environment Variables:
-            ctx      - command invocation context
-            bot      - bot object
+            ctx      - the command invocation context
+            bot      - the bot object
             channel  - the current channel object
-            author   - command author's member object
+            author   - the command author's member object
+            guild    - the current guild object
             message  - the command's message object
-            discord  - discord.py library
-            commands - redbot.core.commands
-            _        - The result of the last dev command.
+            aiohttp  - the aiohttp library
+            asyncio  - the asyncio library
+            discord  - the discord.py library
+            commands - the redbot.core.commands module
+            cf       - the redbot.core.utils.chat_formatting module
+            _        - the result of the last dev command
```

Comment:
The `_` in `chat_formatting` and the `_` env var cause this section to be italicized. The env vars should probably be put in inline code blocks, [see here](https://discord.com/channels/133049272517001216/133251234164375552/1034244615672438834).